hack/lib/vars.bash

```diff
@@ -51,8 +51,8 @@ export OPERATORS_NAMESPACE="${OPERATORS_NAMESPACE:-openshift-serverless}"
 export SERVING_NAMESPACE="${SERVING_NAMESPACE:-knative-serving}"
 export INGRESS_NAMESPACE="${INGRESS_NAMESPACE:-knative-serving-ingress}"
 export EVENTING_NAMESPACE="${EVENTING_NAMESPACE:-knative-eventing}"
-# eventing e2e and conformance tests use a container for tracing tests that has hardcoded `istio-system` in it
-export TRACING_NAMESPACE="${TRACING_NAMESPACE:-istio-system}"
+# eventing e2e and conformance tests use a container for tracing tests that has hardcoded to `knative-tracing` in it
```

Comment:
Has Eventing switched to that or it will pick up the new value on the fly?

Reply:
it is this: https://github.com/openshift-knative/serverless-operator/pull/2892/files#diff-e4501ba318a93735eec86c7e49b3c169968642a24d2966184001c1dcb0a00c0fR30

I could not find another place where that was set/used.

Reply:
cc @pierDipi  @creydr  ^^

Reply:
Also I think Eventing sets the variable on their tests (see Makefile) to `knative-eventing` anyway.

Reply:
Yes, I don't think we have ever used istio components for testing tracing, we deploy zipkin

Reply:
Zipkin used to be installed in istio-system namespace upstream, AFAIK. Maybe the "istio-system" value is not hardcoded in Go files anymore upstream (at least I can't find it now).